pylint/checkers/utils.py

```diff
@@ -50,13 +50,12 @@
 from functools import lru_cache, partial
 from typing import Callable, Dict, Iterable, List, Match, Optional, Set, Tuple, Union
 
+import _string  # pylint: disable=wrong-import-position, wrong-import-order
```

Comment:
Maybe it's time to remove that disable ?

Reply:
Good catch.